storage.js

```diff
@@ -0,0 +1,28 @@
+const userNames = document.getElementById('user_name');
+const lastName = document.getElementById('last_name');
+const useremail = document.getElementById('user_email');
+const userMessage = document.getElementById('user-message');
+
+function setStorage() {
+  const currentName = localStorage.getItem('name');
+  const currentLastName = localStorage.getItem('lastName');
+  const currentEmail = localStorage.getItem('email');
+  const currentMessage = localStorage.getItem('message');
```

Comment:
- [ ] Using the correct method to get the data from the local storage is cool. But it will be nice if you could just do this in just two lines and parse the data from a string format to a JSON object because you know we stringify the data before storing it.

See guide:
```
let savedFormData = localStorage.getItem('data');
savedFormData = JSON.parse(savedFormData);
```